parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -302,7 +302,7 @@ ensureJournal(){
 }
 
 ensurePodSecurityPolicy() {
```

Comment:
I think we can get rid of this `ensurePodSecurityPolicy` func (and anywhere that it is called) altogether now that we're putting it in the `addons/` directory. Manifests under `/etc/kubernetes/addons/` should be loaded automatically by the addon-manager component, so we just need to do all our "is psp enabled?" conditional logic during template generation, but not at provisioning runtime.

Reply:
Great insight. It took me a while to figure it how to proper use the addon-manager (my bad!) but I really liked this approach. Thanks @jackfrancis and @CecileRobertMichon!